packages/podman-extension-api/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I wanted to discuss that with @benoitf, but I feel like it would be easier if the types of the extensions would be in the extension folder itself.

This is not really possible today, as to publish a npm package the `name` property of the `package.json` must be equal to the name of the package. 

However, IMO creating a new package just for the type for each extensions we want to expose an api would be too much... We might want to discuss how we could do this nicely.

I also feel like it is very distrurbing to have 

- `packages/api`
- `packages/extension-api`
- `packages/podman-extension-api`

where it feel messy

Reply:
That is 'internal implementation' we can start with this location and then move packages around. That won't affect the API itself.